src/main/java/uk/gov/ons/census/fwmt/rmadapter/config/GatewayEventsConfig.java

```diff
@@ -10,13 +10,14 @@
   public static final String CANONICAL_CREATE_SENT = "Canonical - Action Create Sent";
   public static final String CANONICAL_UPDATE_SENT = "Canonical - Action Update Sent";
   public static final String CANONICAL_CANCEL_SENT = "Canonical - Action Cancel Sent";
+  public static final String CANONICAL_CANCEL_FAILED ="Canonical - Action cancel failed. Address type is not a Household (HH)";
   public static final String RM_REQUEST_RECEIVED = "RM - Request Received";
 
   @Bean
   public GatewayEventManager gatewayEventManager() {
     GatewayEventManager gatewayEventManager = new GatewayEventManager();
     gatewayEventManager.addEventTypes(new String[] {CANONICAL_CREATE_SENT, CANONICAL_UPDATE_SENT, CANONICAL_CANCEL_SENT,
-        RM_REQUEST_RECEIVED});
+        RM_REQUEST_RECEIVED, CANONICAL_CANCEL_FAILED});
```

Comment:
this canonical cancel failed, does it work as you intended it to? You'll have to explain how you are using it within the event logger.